src/fidesops/ops/service/privacy_request/request_runner_service.py

```diff
@@ -343,20 +354,70 @@ async def run_privacy_request(
         )
         if not proceed:
             return
-
+        if config.notifications.send_request_completion_notification:
+            try:
+                initiate_privacy_request_completion_email(
+                    session, policy, access_result_urls, identity_data
+                )
+            except IdentityNotFoundException as e:
+                logger.error(e)
+                privacy_request.error_processing(db=session)
+                # If dev mode, log traceback
+                await fideslog_graph_failure(
+                    failed_graph_analytics_event(privacy_request, e)
+                )
+                _log_exception(e, config.dev_mode)
+                return
         privacy_request.finished_processing_at = datetime.utcnow()
         AuditLog.create(
             db=session,
             data={
                 "user_id": "system",
                 "privacy_request_id": privacy_request.id,
                 "action": AuditLogAction.finished,
+                # fixme- this was existing code but "finished" isn't an AuditLogAction?
                 "message": "",
             },
         )
         privacy_request.status = PrivacyRequestStatus.complete
-        privacy_request.save(db=session)
         logging.info("Privacy request %s run completed.", privacy_request.id)
+        privacy_request.save(db=session)
+
+
+def initiate_privacy_request_completion_email(
+    session: Session,
+    policy: Policy,
+    access_result_urls: List[str],
+    identity_data: Dict[str, Any],
+) -> None:
+    if policy.get_rules_for_action(action_type=ActionType.access):
```

Comment:
got it, so to clarify, how will I check for any `manual_data` within the `request_runner_service.py`?

Reply:
nothing to do here since my work isn't merged, it might be me that reconciles, but there will be a `manual_data` dictionary created inside `run_privacy_requeset`, so you could just pass that into this same function alongside the access_result_urls.  The manual_data also only exists for access requests.

Reply:
Makes sense, thanks for the heads up!